metallb/metallb-operator
# Metallb Layer 2 config controller

Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>
Metallb L2 config controller 
Unit-Test:
- bringup multi node KIND cluster
- install and run controller (make install , make run)
- apply sample config
- configure NGINX loadbalancer service
- make sure External IP is allocated and curl it
